src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -116,8 +122,8 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
 
   private def createUserInternal(user: SamUser, samRequestContext: SamRequestContext): IO[SamUser] = {
     for {
-      createdUser <- directoryDAO.createUser(user, samRequestContext)
-      _ <- registrationDAO.createUser(user, samRequestContext)
+      createdUser <- directoryDAO.createUser(user, samRequestContext).withDebugLogMessage(s"Creating user with id ${user.toUserIdInfo.userSubjectId} in directory")
```

Comment:
```suggestion
      createdUser <- directoryDAO.createUser(user, samRequestContext).withDebugLogMessage(s"Creating user with id ${user.toUserIdInfo.userSubjectId}")
```

I think the simpler message is OK here.  Postgres is the source of record and so shouldn't need the additional location specified.  Subsequent line stating LDAP is good though, that makes sense and will hopefully go away soon.